grapher/Models/Calculations/AccelCalculator.cs

```diff
@@ -203,6 +204,8 @@ public void CalculateCombinedDiffSens(AccelData data, ManagedAccel accel, Driver
 
                 var xRatio = settings.sensitivity.x * ratio;
                 var yRatio = settings.sensitivity.y * ratio;
+                xRatio = Math.Min(int.MaxValue, Math.Max(int.MinValue, xRatio));
+                yRatio = Math.Min(int.MaxValue, Math.Max(int.MinValue, yRatio));
```

Comment:
Can we make this a helper named something like "ClampToInt"?

Reply:
Maybe, but unless we even know, clamping to int really makes sense (not that it just works for these two extreme cases), it's not worth polishing.